web/cobrands/bristol/assets.js

```diff
@@ -4,27 +4,62 @@ if (!fixmystreet.maps) {
     return;
 }
 
-// Assets are served from two different WFS feeds; one for lighting and one
-// for everything else. They have some options in common:
-var options = {
+var base_options = {
     max_resolution: {
-        'bristol': 0.33072982812632296,
-        'fixmystreet': 4.777314267158508
+	'bristol': 0.33072982812632296,
+	'fixmystreet': 4.777314267158508
     },
-    asset_type: 'spot',
     body: "Bristol City Council",
-    srsName: "EPSG:27700",
     geometryName: 'SHAPE',
```

Comment:
No need for final comma in object (okay, doesn't break anything nowadays but still, might as well).